src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -1651,6 +1663,9 @@
   <data name="ArgumentOutOfRange_DateTimeBadTicks" xml:space="preserve">
     <value>Ticks must be between DateTime.MinValue.Ticks and DateTime.MaxValue.Ticks.</value>
   </data>
+  <data name="ArgumentOutOfRange_TimeOnlyBadTicks" xml:space="preserve">
+    <value>Ticks must be between 0 and 863_999_999_999.</value>
```

Comment:
Do we use underscores elsewhere as digit separators?  I'd have expected commas and then that is something that would be localized as needed.

Reply:
Also, existing resource strings like ArgumentOutOfRange_DateTimeBadTicks above use named constants rather than the numerical value.  Could this instead be `and TimeOnly.MaxValue.Ticks`?

Reply:
That is better idea. I'll use `TimeOnly.MaxValue.Ticks`